### PR TITLE
applied PackageJanitor

### PR DESCRIPTION
**Pull request by mohamed-barakat**

### CODE REVIEW

File: examples/TerminalCategory.g [hunk 1]
```diff
@@ -8,7 +8,7 @@ T := FiniteCompletion( InitialCategory( ) );
 IsTerminalCategory( T );
 #! true
 InfoOfInstalledOperationsOfCategory( T );
-#! 492 primitive operations were used to derive 524 operations for this category
+#! 496 primitive operations were used to derive 524 operations for this category
```
Comment (zickgraf) [inline examples/TerminalCategory.g]: I still get 492 operations locally, even with CAP 2022.08-09. Maybe some other package is responsible?
```
gap> LoadPackage("FunctorCategories");
Loading AutoDoc 2022.07.10 (Generate documentation from GAP source code)
Loading AutPGrp 1.10.2 (Computing the Automorphism Group of a p-Group)
Loading Browse 1.8.9 (ncurses interface and browsing applications)
Loading Gauss 2022.08-04 (Extended Gauss functionality for GAP)
Loading ToolsForHomalg 2022.08-02 (Special methods and knowledge propagation tools)
Loading homalg 2022.08-04 (A homological algebra meta-package for computable Abelian categories)
Loading Alnuth 3.1.2 (Algebraic number theory and an interface to PARI/GP)
Loading Polycyclic 2.15.1 (Computation with polycyclic groups)
Loading Utils 0.69 for GAP 4.10 - a collection of utility functions.
Loading MatricesForHomalg 2022.08-04 (Matrices for the homalg project)
Loading GaussForHomalg 2022.08-02 (Gauss functionality for the homalg project)
Loading HomalgToCAS 2022.08-02 (A window to the outer world)
Loading Modules 2022.08-03 (A homalg based package for the Abelian category of finitely presented modules over computable rings)
Loading IO_ForHomalg 2022.08-03 (IO capabilities for the homalg project)
Loading RingsForHomalg 2022.08-04 (Dictionaries of external rings)
Loading GradedRingForHomalg 2022.08-02 (Endow Commutative Rings with an Abelian Grading)
Loading CAP 2022.08-09 (Categories, Algorithms, Programming)
Loading CompilerForCAP 2022.08-10 (Speed up computations in CAP categories)
Loading MonoidalCategories 2022.08-04 (Monoidal and monoidal (co)closed categories)
Loading ToolsForHigherHomologicalAlgebra 2022.08-01 (Tools for the Higher Homological Algebra project)
Loading GeneralizedMorphismsForCAP 2022.08-01 (Implementations of generalized morphisms for the CAP project)
Loading CategoryConstructor 2022.06-04 (Construct categories out of given ones)
Loading CartesianCategories 2022.08-03 (Cartesian and cocartesian categories and various subdoctrines)
Loading ComplexesCategories 2022.08-02 (Category of (co)chain complexes of an additive category)
Loading Toposes 2022.08-01 (Elementary toposes)
Loading QPA 2.0-dev (Quivers and Path Algebras)
Loading FinSetsForCAP 2022.08-02 (The elementary topos of (skeletal) finite sets)
Loading FreydCategoriesForCAP 2022.08-11 (Freyd categories - Formal (co)kernels for additive categories)
Loading LinearAlgebraForCAP 2022.08-05 (Category of Matrices over a Field for CAP)
Loading SubcategoriesForCAP 2022.08-08 (Subcategory and other related constructors for CAP categories)
Loading Algebroids 2022.08-03 (Algebroids and bialgebroids as preadditive categories generated by enhanced quivers)
Loading FunctorCategories 2022.08-03 (Categories of functors)
true
gap> T := FiniteCompletion( InitialCategory( ) );
FiniteCompletion( InitialCategory( ) )
gap> InfoOfInstalledOperationsOfCategory( T );
492 primitive operations were used to derive 524 operations for this category which constructively
* IsEquippedWithHomomorphismStructure
* IsLinearCategoryOverCommutativeRing
* IsCodistributiveCocartesianCategory
* IsBicartesianClosedCategory
* IsBicartesianCoclosedCategory
* IsAbelianCategoryWithEnoughInjectives
* IsAbelianCategoryWithEnoughProjectives
* IsRigidSymmetricClosedMonoidalCategory
* IsRigidSymmetricCoclosedMonoidalCategory
```

Comment (zickgraf) [reply examples/TerminalCategory.g]: Ah, there are again instabilities with reversed recnames.

Comment (zickgraf) [reply examples/TerminalCategory.g]: `CartesianBraiding`, `CocartesianBraiding` and their WithGiven versions are only installed with reversed recnames.

Comment (mohamed-barakat) [reply examples/TerminalCategory.g]: > Ah, there are again instabilities with reversed recnames.

Yessss :)

Comment (zickgraf) [reply examples/TerminalCategory.g]: Should be resolved by https://github.com/homalg-project/CAP_project/pull/1004.

Comment (mohamed-barakat) [reply examples/TerminalCategory.g]: Thanks :)

**Pull request merged**